ngmix/gmix.py

```diff
@@ -958,12 +970,14 @@ def _fill(self, pars):
         self._fill_func(
             gm,
             self._pars,
+            self.TdByTe,
```

Comment:
You used `self._TdbyTe` in GMixCM. Not sure if you want to be consistent here.

Reply:
will fix